src/styles/_catalog.scss

```diff
@@ -1,22 +1,38 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  justify-content: center;
+  gap: 48px;
   max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 448px) {
+    grid-template-columns: repeat(2, 300px);
```

Comment:
it is better to define a width variable with 200px value, and use it instead of 200 px